### PR TITLE
Migrate Watchlists visual composer alerts to design system

### PR DESCRIPTION
**Pull request by rmusser01**
## Summary
- Replace VisualComposerPane AntD Alert callouts with the shared design-system Alert primitive.
- Add focused Watchlists coverage for empty composer, section-generation error, and section-generation warning Alert states.
- Remove the three migrated VisualComposerPane Alert entries from the product-state baseline and record TASK-45.44.3.5.

## Verification
- bunx vitest run src/components/Option/Watchlists/TemplatesTab/__tests__/VisualComposerPane.section-generation.test.tsx --maxWorkers=1 --no-file-parallelism
- bunx vitest run src/design-system/__tests__/product-state-guard.test.ts --maxWorkers=1 --no-file-parallelism
- bun run verify:design-system-state
- git diff --check

## Notes
- Bandit skipped: frontend TS/TSX, JSON, and Backlog task metadata only.
- Merge gate: AI-authored PRs still need the requester's human-written Change summary before merge.

---
## Summary by cubic
Migrates Watchlists Visual Composer alerts from `antd` to the design-system `Alert` to unify UI, improve accessibility, and clear product-state exceptions. Completes TASK-45.44.3.5 with no user-visible changes to empty, error, or warning callouts.

- **Refactors**
  - Replace `antd` Alert with `@/components/ui/primitives/Alert` in `VisualComposerPane` (use `variant="info|error|warning"`, keep titles; move empty-state message to children; set `aria-live="off"` on the static info callout).
  - Add focused tests to assert design-system `Alert` usage via `[data-ds-component="Alert"]` for empty, error, and warning states; verify empty-state copy and `aria-live="off"`; confirm warnings render joined by "; ".
  - Remove the three migrated entries from `design-system-product-state-baseline.json`.

<sup>Written for commit e9f84d6bb06e321657e3b571513c6184d238e23c. Summary will update on new commits. <a href="https://cubic.dev/pr/rmusser01/tldw_server/pull/1975?utm_source=github">Review in cubic</a></sup>

### CODE REVIEW

File: apps/packages/ui/src/components/Option/Watchlists/TemplatesTab/VisualComposerPane.tsx [hunk 1]
```diff
@@ -201,11 +202,11 @@ export const VisualComposerPane: React.FC<VisualComposerPaneProps> = ({
...
         <Alert
-          type="info"
-          showIcon
+          variant="info"
           title="No visual blocks yet"
-          description="Add blocks above to start building this template."
-        />
+        >
+          Add blocks above to start building this template.
+        </Alert>
```
[the remark this answers is not in the document]
Comment (rmusser01) [reply apps/packages/ui/src/components/Option/Watchlists/TemplatesTab/VisualComposerPane.tsx:210]: Fixed in 781e4f037: the static empty composer Alert now passes aria-live="off", and the focused VisualComposerPane test asserts that opt-out. Verification reran the focused suite, product-state guard, full design-system verifier, baseline parse, diff check, and TypeScript debt check.
